RedHatInsights/ocp-advisor-frontend
# feat(Recs list): Add risk of change column

Implements https://issues.redhat.com/browse/OCPADVISOR-12.

Must be merged after https://github.com/RedHatInsights/ocp-advisor-frontend/pull/498.

This adds a new Risk of change column to /recommendations table. Also, it enables sorting and filtering by this column.

## How to test

Open /openshift/insights/advisor/recommendations. Check the the new column is present and you can filter and sort by its values. URL parameters must also change. You can also verify that the `resolution_risk` value from API `GET /rule` response is correctly mapped to the appropriate labels in the column.

## Screenshots

![image](https://user-images.githubusercontent.com/31385370/212680874-a83b77fc-66c0-4141-a211-e2c7c07c6dd9.png)
